src/main/java/net/sf/jabref/external/AttachFileAction.java

```diff
@@ -25,7 +26,8 @@ public AttachFileAction(BasePanel panel) {
     @Override
     public void action() {
         if (panel.getSelectedEntries().size() != 1) {
-            return; // TODO: display error message?
+            panel.output(Localization.lang("No entries or multiple entries selected."));
```

Comment:
We should look over that in general. The next action (in the BasePanel (?) code) has exactly the same message. There are also a few related translations strings. For example ~"You must select exactly one entry".

Also, one can consider if a dialog should be shown, possibly with a "do not show this again check box", for this and many other issues. Drawback is that the check box selection must be stored somehow, but maybe one can create a dedicated dialog type for that?